articles/hdinsight/hdinsight-component-versioning.md

```diff
@@ -16,7 +16,7 @@ This table lists the versions of HDInsight that are available in the Azure porta
 
 | HDInsight version | VM OS | Release date| Support type | Support expiration date | Retirement date | High availability |
 | --- | --- | --- | --- | --- | --- | ---|
-| [HDInsight 4.0](hdinsight-40-component-versioning.md) |Ubuntu 18.0.4 LTS |September 24, 2018 | [Standard](hdinsight-component-versioning.md#support-options-for-hdinsight-versions) | | |Yes |
+| [HDInsight 4.0](hdinsight-40-component-versioning.md) |Ubuntu 18.0.4 LTS |September 24, 2018 | [Standard](hdinsight-component-versioning.md#support-options-for-hdinsight-versions) | see https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-component-versioning for cluster type details. | see https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-component-versioning for cluster type details. |Yes |
```

Comment:
I don't understand the proposal. It seems to have a link pointing back to the same page (hdinsight-component-versioning).